Dockerfile

```diff
@@ -54,7 +54,7 @@ RUN set -x && \
         armel) s6Arch='arm' ;; \
         armhf) s6Arch='armhf' ;; \
 		arm64) s6Arch='aarch64' ;; \
-		ppc64le) s6Arch='ppc64le' ;; \
+		ppc64el) s6Arch='ppc64le' ;; \
```

Comment:
Have a peek at this: https://lists.debian.org/debian-powerpc/2014/08/msg00042.html

Reply:
This is what QEMU in Github Actions is capable of doing:

````
| 2021/04/25 21:59:50 installing: mips64el qemu-mips64el already registered
| {
|   "supported": [
|     "linux/amd64",
|     "linux/arm64",
|     "linux/riscv64",
|     "linux/ppc64le",
|     "linux/s390x",
|     "linux/386",
|     "linux/mips64le",
|     "linux/mips64",
|     "linux/arm/v7",
|     "linux/arm/v6"
|   ],
|   "emulators": [
|     "CLR",
|     "qemu-aarch64",
|     "qemu-arm",
|     "qemu-mips64",
|     "qemu-mips64el",
|     "qemu-ppc64le",
|     "qemu-riscv64",
|     "qemu-s390x"
|   ]
| }
````

Reply:
> This is what QEMU in Github Actions is capable of doing:
> 
> ```
> | 2021/04/25 21:59:50 installing: mips64el qemu-mips64el already registered
> | {
> |   "supported": [
> |     "linux/amd64",
> |     "linux/arm64",
> |     "linux/riscv64",
> |     "linux/ppc64le",
> |     "linux/s390x",
> |     "linux/386",
> |     "linux/mips64le",
> |     "linux/mips64",
> |     "linux/arm/v7",
> |     "linux/arm/v6"
> |   ],
> |   "emulators": [
> |     "CLR",
> |     "qemu-aarch64",
> |     "qemu-arm",
> |     "qemu-mips64",
> |     "qemu-mips64el",
> |     "qemu-ppc64le",
> |     "qemu-riscv64",
> |     "qemu-s390x"
> |   ]
> | }
> ```

Problem not with Github QEMU emulator, but with debian 'dpkg --print-architecture'. It report ppc64**el** on ppc64le QEMU emulator.